samples/java/README.md

```diff
@@ -0,0 +1,58 @@
+# Lead Integration - Java
```

Comment:
How about creating a ptbr version of readme? 


`https://github.com/olxbr/crm-lead-integration/blob/master/samples/nodejs/README-ptbr.md`

Reply:
Added ptbr version:

`samples/java/README-ptbr.md`